task/05-regex-tasks.js

```diff
@@ -72,7 +73,7 @@ function getRegexForPitSpot() {
  * @return {RegExp}
  */
 function getRegexForIPv4() {
-   throw new Error('Not implemented');
+    return /^([0-9]|[0-9][0-9]|[01][0-9][0-9]|2[0-4][0-9]|25[0-5])(\.([0-9]|[0-9][0-9]|[01][0-9][0-9]|2[0-4][0-9]|25[0-5])){3}$/;
```

Comment:
You can do it without repeating  the range part (0-255). Hint: use negative lookahead, nested groupts and notice that you can put $ at any place in your regex. This regex is good though. Had i seen this in production i wouldn't ask to fix it. But for a educational purpose try to do it without repeating.

Reply:
my logic is to take all 3 digits numbers and exclude from match numbers from [256 to 999]
![screenshot from 2017-09-19 12-37-44](https://user-images.githubusercontent.com/26328278/30586143-a8d6ae6e-9d37-11e7-93d3-0b47b0b0bb7a.png)

/^(?!([2-9][6-9][0-9]|[3-9][0-9][0-9]))([0-9]|[0-9][0-9]|[0-9][0-9][0-9])(\.([0-9]|[0-9][0-9]|[0-9][0-9][0-9])){3}$/